test/all-templates.test.js

```diff
@@ -1,32 +1,138 @@
 'use strict';
 
-// This file contains Jest tests that run against every function template in
-// the repo.
+// These tests  verify that some important guarantees hold for all the functions
+// in the repository.
 
 const path = require('path');
 const fs = require('fs');
+const { parser } = require('configure-env');
 
-const excludedPaths = ['node_modules', 'test', 'coverage', 'docs'];
+// skipList is a list of function templates that don't pass verification
+// for now, but will in the long term
+const skipList = ['conversations', 'funlet-call-me', 'funlet-echo',
+                  'funlet-find-me', 'funlet-forward', 'funlet-simple-menu',
+                  'funlet-simple-message', 'funlet-simulring',
+                  'funlet-whisper', 'vaccine-standby'];
+const excludedPaths = ['node_modules', 'test', 'coverage', 'docs', 'blank'] + skipList;
 const projectRoot = path.resolve(__dirname, '..');
 // Assemble a list of template directories here, since templates.json
 // may not have all of them:
 const templates = fs
       .readdirSync(projectRoot, { withFileTypes: true })
       .filter(
-        (file) =>
-        file.isDirectory() &&
-          !file.name.startsWith('.') &&
-          !file.name.startsWith('_') &&
-          !excludedPaths.includes(file.name)
+          (file) =>
+          file.isDirectory() &&
+              !file.name.startsWith('.') &&
+              !file.name.startsWith('_') &&
+              !excludedPaths.includes(file.name)
```

Comment:
Seems like the indentation in this file is 4 and I think the rest of the project should be two if I'm not mistaken.